### PR TITLE
Test for reposync --delete with multiple repos

### PR DESCRIPTION
**Pull request by m-blaha**
Requires: https://github.com/rpm-software-management/dnf-plugins-core/pull/374

### CODE REVIEW

File: dnf-behave-tests/features/plugins-core/reposync.feature [hunk 1]
```diff
@@ -167,6 +167,15 @@ Scenario: Reposync downloads packages and removes packages that are not part of
     And file "//{context.dnf.tempdir}/setopt.ext/x86_64/flac-libs-1.0-1.fc29.x86_64.rpm" does not exist
 
 
+@bz1774103
+Scenario: Reposync --delete does not immediately delete downloaded content when multiple repositories are synced
+  Given I use repository "reposync" as http
+    And I use repository "dnf-ci-thirdparty-updates" as http
+   When I execute dnf with args "reposync --download-path={context.dnf.tempdir} --delete"
+   Then the exit code is 0
+    And stdout does not contain "[DELETED]"
```
Comment (pkratoch) [inline dnf-behave-tests/features/plugins-core/reposync.feature]: Can you please also check the presence of the files? (If we decide to change output from "[DELETED]" to something else, all reposync.feature tests would pass anyway.)

Comment (m-blaha) [reply dnf-behave-tests/features/plugins-core/reposync.feature]: That's a good point. Fixed.

Comment (pkratoch) [reply dnf-behave-tests/features/plugins-core/reposync.feature]: Thanks. You missed "//{context.dnf.tempdir}/reposync/src/wget-1.0-1.fc29.src.rpm", though.

Comment (m-blaha) [reply dnf-behave-tests/features/plugins-core/reposync.feature]: Fixed.

**Pull request merged**